k8s/vagrant/README.md

```diff
@@ -1,27 +1,36 @@
 # Installing Kubernetes Clusters on Ubuntu 16.04 using Vagrant
 
-OpenEBS provides vagrant boxes with prepackaged kuberentes images. There are different vagrant boxes created depending on the kubernetes release. The Vagrantfiles are organized here based on the kubernetes version used by the box. A deployer can choose the Vagrantfile, based on his choice of kubernetes version. 
+OpenEBS provides vagrant boxes with prepackaged kuberentes images. There are different vagrant boxes created depending on the kubernetes release. The Vagrantfiles are organized here based on the kubernetes version used by the box. 
 
-This Vagrantfile can be used on laptop or Baremetal server installed with Ubuntu 16.04 and Virtualization Enabled
+This Vagrantfile can be used on any machine with Virtualization Enabled, like laptop or baremetal server. 
 
+The below instructions will walk you through the following:
+- Verify Prerequisites
+- Download Vagrantfile
+- Bringup K8s Cluster
+- Install kubectl on the host 
+- Setup access to Kubernetes UI/Dashboard (Starting from Vagrantfile 1.7.5)
+- Setup OpenEBS
+- Launch demo pod
+
+*Note: The instructions are from an Ubuntu 16.06 host.*
 
 ## Prerequisites
 
 Verify that you have the following required software installed on your Ubuntu 16.04 machine:
 ```
 1.Vagrant (>=1.9.1)
 2.VirtualBox 5.1
-3.curl or wget or git, etc., to download the Vagrant file. 
 ```
 
 ## Download and Verify 
 
-Dowload the required Vagrantfile. You can use wget to download a specific vagrant file, like shown below - or you can just close the repository locally. 
+Download the required Vagrantfile. Use curl or wget or git, etc., to download the required Vagrantfile. This example uses wget.
 
 ```
 mkdir k8s-demo
 cd k8s-demo
-wget https://raw.githubusercontent.com/openebs/openebs/master/k8s/vagrant/1.6/Vagrantfile
+wget https://raw.githubusercontent.com/openebs/openebs/master/k8s/vagrant/1.7.5/Vagrantfile
 vagrant status
```

Comment:
Vagrant Status

Reply:
the command is *vagrant status* -- with lowercase "v". Leaving it as is.